src/modules/uORB/uORBDeviceNode.cpp

```diff
@@ -161,6 +161,75 @@ uORB::DeviceNode::close(cdev::file_t *filp)
 	return CDev::close(filp);
 }
 
+bool
+uORB::DeviceNode::copy_locked(void *dst, unsigned &generation)
+{
+	bool updated = false;
+
+	if ((dst != nullptr) && (_data != nullptr)) {
```

Comment:
Using an early return would reduce the level of indentation:
```suggestion
	if (!dst || !_data) {
	   return false;
	}
```